samples/drivers/lora/send/README.rst

```diff
@@ -0,0 +1,36 @@
+.. zephyr:code-sample:: lora
+   :name: lora send
+   :relevant-api: lora_radio
+
+   Transmit a preconfigured payload every second using the lora radio.
+
+Overview
+********
+
+This sample demonstrates how to use the lora radio driver to configure
+the encoding settings and send data over the radio.
+
+Transmitted messages can be received by building and flashing the accompanying
+lora receive sample on another board.
+
+Building and Running
+********************
+
+Build and flash the sample as follows, changing ``b_l072z_lrwan1`` for
+your board, where you board has a default lora radio.
```

Comment:
> default lora radio

I think it is worth briefly expanding on what this means (has a `lora0` alias in devicetree)

Reply:
Understood, I have added your suggestion.